examples/chart/teleport-cluster/values.yaml

```diff
@@ -27,6 +27,16 @@ authenticationType: local
 # Possible values are 'multiplex'
 proxyListenerMode: ""
 
+# Optional setting for configuring session recording. Possible values are:
+#    "node"  : sessions will be recorded on the node level  (the default)
+#    "node-sync" : session recordings will be streamed from node -> auth -> storage service without being stored on disk at all.
```

Comment:
Is "storage service" the same as our [storage backends](https://goteleport.com/docs/setup/reference/backends/) in this context? If so, we should probably say "storage backend" so our docs are consistent.

Reply:
This is copied from https://github.com/gravitational/teleport/blob/master/docs/pages/setup/reference/config.mdx

Storage service is typically not the same as the storage backend afaik. At least for the GCP and AWS deployments, the storage backends are Firestore and DynamoDB and the storage service used for session recording is a GCS and S3 bucket respectively.

From that link the "storage service" referenced here is the "session recordings" data type.